lib/warder/bundle_audit_runner.rb

```diff
@@ -14,8 +14,19 @@ def command_with_options
       "#{COMMAND_NAME} update; (cd #{path} && #{COMMAND_NAME} check)"
     end
 
+    def printable?(line)
+      super && !FAILURE_REGEXP.match(line)
+    end
+
     def failed?(line)
-      FAILURE_REGEXP.match(line)
+      match = FAILURE_REGEXP.match(line)
+      return false unless match
+      @stats_msg = line
+      !match[1].match('No ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.